src/main/java/edu/cornell/kfs/rass/batch/xml/RassXmlDocumentWrapper.java

```diff
@@ -0,0 +1,88 @@
+package edu.cornell.kfs.rass.batch.xml;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Objects;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlElementWrapper;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import edu.cornell.kfs.sys.xmladapters.RassStringToJavaLongDateTimeAdapter;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlRootElement(name = "Kfs", namespace = StringUtils.EMPTY)
+public class RassXmlDocumentWrapper {
+    
+    @XmlElement(name = "Extract_Begin_Timestamp", namespace = StringUtils.EMPTY)
+    @XmlJavaTypeAdapter(RassStringToJavaLongDateTimeAdapter.class)
+    private Date extractDate;
+    
+    @XmlElementWrapper(name = "Awards", namespace = StringUtils.EMPTY)
+    @XmlElement(name = "Award", namespace = StringUtils.EMPTY, required = true)
```

Comment:
There a few annotations in this class that still have the "required" attribute on them.

Reply:
ah, thanks, I shall fix